project/api/views.py

```diff
@@ -125,6 +129,111 @@ class SpacetimeVolumeViewSet(viewsets.ModelViewSet):
     )
     serializer_class = SpacetimeVolumeSerializer
 
+    @transaction.atomic
+    def create(self, request, *args, **kwargs):
+        """
+        Solve overlaps if included in request body
+        """
+
+        if not issubclass(type(request.data["territory"]), UploadedFile):
+            print(
+                "Territory type {}, Content {}".format(
+                    type(request.data["territory"]), request.data["territory"]
+                )
+            )
+            raise ValidationError("territory file is missing")
+
+        print(
+            ">🗺️ STV Filename: {}, Size: {}MB".format(
+                request.data["territory"],
+                round(request.data["territory"].size / 1048576, 2),
+            )
+        )
+
+        content = request.data["territory"].read().decode("utf-8")
+        geom = GEOSGeometry(content)
```

Comment:
This one should be handled separately, because we want to notify user, that content of his file is not valid

Reply:
As my GeoJSON can't pass the GEOSGeometry validation, I will show the response from it here
This is my code block

```suggestion
        try:
            geom = GEOSGeometry(content)
        except GDALException:
            rmsg = "Geometry is not recognized"
            raise ValidationError('{"error": %(error)s}', params={"error": rmsg})
```
So I do expect a JSON response, with error code and error message in params, but instead I'am getting a debug page


```
ValidationError at /api/spacetime-volumes/
['{"error": Geometry is not recognized}']

Request Method: POST
Request URL: http://web/api/spacetime-volumes/
Django Version: 2.2.7
Python Executable: /usr/local/bin/python
Python Version: 3.8.0
Python Path: ['/src', '/opt/python/bin', '/usr/local/lib/python38.zip', '/usr/local/lib/python3.8', '/usr/local/lib/python3.8/lib-dynload', '/opt/python/lib/python3.8/site-packages', '/usr/local/lib/python3.8/site-packages']
Server time: Tue, 28 Jan 2020 08:01:38 +0000
Installed Applications:
['django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.gis',
 'django.contrib.postgres',
 'django_extensions',
 'api.apps.ApiConfig',
 'colorfield',
 'rest_framework',
 'rest_framework_gis',
 'drf_firebase_auth',
 'ordered_model',
 'silk',
 'debug_toolbar',
 'django.contrib.admin',
 'django.contrib.admindocs']
Installed Middleware:
['django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware',
 'silk.middleware.SilkyMiddleware']


Traceback:

File "/src/api/views.py" in create
  157.             geom = GEOSGeometry(content)

File "/opt/python/lib/python3.8/site-packages/django/contrib/gis/geos/geometry.py" in __init__
  711.                 ogr = gdal.OGRGeometry.from_json(geo_input)

File "/opt/python/lib/python3.8/site-packages/django/contrib/gis/gdal/geometries.py" in from_json
  161.         return OGRGeometry(OGRGeometry._from_json(force_bytes(geom_input)))

File "/opt/python/lib/python3.8/site-packages/django/contrib/gis/gdal/geometries.py" in _from_json
  143.         ptr = capi.from_json(geom_input)

File "/opt/python/lib/python3.8/site-packages/django/contrib/gis/gdal/prototypes/errcheck.py" in check_geom
  84.         raise GDALException('Invalid geometry pointer returned from "%s".' % func.__name__)

During handling of the above exception (Invalid geometry pointer returned from "OGR_G_CreateGeometryFromJson".), another exception occurred:

File "/opt/python/lib/python3.8/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/opt/python/lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/opt/python/lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/opt/python/lib/python3.8/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/opt/python/lib/python3.8/site-packages/rest_framework/viewsets.py" in view
  114.             return self.dispatch(request, *args, **kwargs)

File "/opt/python/lib/python3.8/site-packages/rest_framework/views.py" in dispatch
  505.             response = self.handle_exception(exc)

File "/opt/python/lib/python3.8/site-packages/rest_framework/views.py" in handle_exception
  465.             self.raise_uncaught_exception(exc)

File "/opt/python/lib/python3.8/site-packages/rest_framework/views.py" in raise_uncaught_exception
  476.         raise exc

File "/opt/python/lib/python3.8/site-packages/rest_framework/views.py" in dispatch
  502.             response = handler(request, *args, **kwargs)

File "/usr/local/lib/python3.8/contextlib.py" in inner
  75.                 return func(*args, **kwds)

File "/src/api/views.py" in create
  160.             raise ValidationError('{"error": %(error)s}', params={"error": rmsg})

Exception Type: ValidationError at /api/spacetime-volumes/
Exception Value: ['{"error": Geometry is not recognized}']
Request information:
USER: Michael_Orlov

GET: No GET data

POST:
end_date = '2458849'
start_date = '2458485'

FILES:
territory = <InMemoryUploadedFile: Estonia_1919-1-1_1940-12-31.json (application/json)>

COOKIES: No cookie data

META:
CONTENT_LENGTH = '99005'
CONTENT_TYPE = 'multipart/form-data; boundary=----WebKitFormBoundaryg4mhv6FhSo58ZkBR'
HTTP_ACCEPT = 'application/json, text/plain, */*'
HTTP_ACCEPT_ENCODING = 'gzip, deflate, br'
HTTP_ACCEPT_LANGUAGE = 'en-GB,en;q=0.9,ru-RU;q=0.8,ru;q=0.7,en-US;q=0.6'
HTTP_AUTHORIZATION = 'JWT eyJhbGciOiJSUzI1NiIsImtpZCI6IjI1OTc0MmQyNjlhY2IzNWZiNjU3YzBjNGRkMmM3YjcyYWEzMTRiNTAiLCJ0eXAiOiJKV1QifQ.eyJuYW1lIjoiTWljaGFlbCBPcmxvdiIsInBpY3R1cmUiOiJodHRwczovL2xoNC5nb29nbGV1c2VyY29udGVudC5jb20vLWRpLV82dUdoZHc4L0FBQUFBQUFBQUFJL0FBQUFBQUFBQ1JZL2h1QzcyV1ppNEpVL3Bob3RvLmpwZyIsImlzcyI6Imh0dHBzOi8vc2VjdXJldG9rZW4uZ29vZ2xlLmNvbS9jaHJvbm1hcHNjb20tZGV2IiwiYXVkIjoiY2hyb25tYXBzY29tLWRldiIsImF1dGhfdGltZSI6MTU3OTcwMjUxNSwidXNlcl9pZCI6IlRsSUdxcXR3dnRRM0lxdTFhd0NKSXNTdHVPOTIiLCJzdWIiOiJUbElHcXF0d3Z0UTNJcXUxYXdDSklzU3R1TzkyIiwiaWF0IjoxNTgwMTk4NDk2LCJleHAiOjE1ODAyMDIwOTYsImVtYWlsIjoibWlrbGVyZ21AZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnsiZ29vZ2xlLmNvbSI6WyIxMDk2NTMxMDcyNjQ2MDIxNTY2MjkiXSwiZW1haWwiOlsibWlrbGVyZ21AZ21haWwuY29tIl19LCJzaWduX2luX3Byb3ZpZGVyIjoiZ29vZ2xlLmNvbSJ9fQ.AXRbHZpE_94Rfj9u3B5zmBI9bJw80vz_4Lw39LYZJ3S849Ly398GcZvIDmhtw9lL4r574LhDdBU_Ye6n2MCugicJlLwOkgi7q7SideP6LLQraKYSlOu5OV-6Zr9n9nTwFO3L9_WWFOynhFrF9gRIz2f2XgFChE6A4jwmWsEAe0Evx40hMxJSrd0c-YU2c0dXhahfbyurK6KsQdQe-Z-ryK1-Ja-m4RpyHc4vUsnMLB-KwzCutWImMEeCnfOnaAzvLq6qp0iwFHfOrxnxycE9-W3idLh55lSvdrVP3mbr8M59zMXfuqx3C9n3LwwVTKfMDerIBGEFR9YUqOQisblYjg'
HTTP_CACHE_CONTROL = 'no-cache'
HTTP_CONNECTION = 'close'
HTTP_DNT = '1'
HTTP_HOST = 'web'
HTTP_ORIGIN = 'http://localhost:3000'
HTTP_PRAGMA = 'no-cache'
HTTP_REFERER = 'http://localhost:3000/admin/te/8'
HTTP_SEC_FETCH_MODE = 'cors'
HTTP_SEC_FETCH_SITE = 'same-origin'
HTTP_USER_AGENT = 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) snap Chromium/79.0.3945.117 Chrome/79.0.3945.117 Safari/537.36'
PATH_INFO = '/api/spacetime-volumes/'
QUERY_STRING = ''
RAW_URI = '/api/spacetime-volumes/'
REMOTE_ADDR = '172.19.0.6'
REMOTE_PORT = '34338'
REQUEST_METHOD = 'POST'
SCRIPT_NAME = ''
SERVER_NAME = '0.0.0.0'
SERVER_PORT = '80'
SERVER_PROTOCOL = 'HTTP/1.0'
SERVER_SOFTWARE = 'gunicorn/19.7.1'
gunicorn.socket = <socket.socket fd=9, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('172.19.0.5', 80), raddr=('172.19.0.6', 34338)>
wsgi.errors = <gunicorn.http.wsgi.WSGIErrorsWrapper object at 0x7f0868402a30>
wsgi.file_wrapper = ''
wsgi.input = <gunicorn.http.body.Body object at 0x7f0868530d30>
wsgi.multiprocess = False
wsgi.multithread = False
wsgi.run_once = False
wsgi.url_scheme = 'http'
wsgi.version = '(1, 0)'

Settings:
Using settings module chron.settings
ABSOLUTE_URL_OVERRIDES = {}
ADMINS = []
ALLOWED_HOSTS = ['web', '127.0.0.1', 'localhost']
APPEND_SLASH = True
AUTHENTICATION_BACKENDS = ['django.contrib.auth.backends.ModelBackend']
AUTH_PASSWORD_VALIDATORS = '********************'
AUTH_USER_MODEL = 'auth.User'
BASE_DIR = '/src'
CACHEOPS = {'api.*': '********************'}
CACHEOPS_DEFAULTS = {'timeout': 86400}
CACHEOPS_REDIS = 'redis://redis:6379/0'
CACHES = {'default': {'BACKEND': 'django_redis.cache.RedisCache', 'LOCATION': 'redis://redis:6379/0', 'OPTIONS': {'CLIENT_CLASS': 'django_redis.client.DefaultClient'}}}
CACHE_MIDDLEWARE_ALIAS = 'default'
CACHE_MIDDLEWARE_KEY_PREFIX = '********************'
CACHE_MIDDLEWARE_SECONDS = 600
CSRF_COOKIE_AGE = 31449600
CSRF_COOKIE_DOMAIN = None
CSRF_COOKIE_HTTPONLY = False
CSRF_COOKIE_NAME = 'csrftoken'
CSRF_COOKIE_PATH = '/'
CSRF_COOKIE_SAMESITE = 'Lax'
CSRF_COOKIE_SECURE = False
CSRF_FAILURE_VIEW = 'django.views.csrf.csrf_failure'
CSRF_HEADER_NAME = 'HTTP_X_CSRFTOKEN'
CSRF_TRUSTED_ORIGINS = []
CSRF_USE_SESSIONS = False
DATABASES = {'default': {'ENGINE': 'django.contrib.gis.db.backends.postgis', 'NAME': 'postgres', 'USER': 'postgres', 'PASSWORD': '********************', 'HOST': 'db', 'PORT': 5432, 'ATOMIC_REQUESTS': False, 'AUTOCOMMIT': True, 'CONN_MAX_AGE': 0, 'OPTIONS': {}, 'TIME_ZONE': None, 'TEST': {'CHARSET': None, 'COLLATION': None, 'NAME': None, 'MIRROR': None}}}
DATABASE_ROUTERS = []
DATA_UPLOAD_MAX_MEMORY_SIZE = 52428800
DATA_UPLOAD_MAX_NUMBER_FIELDS = 1000
DATETIME_FORMAT = 'N j, Y, P'
DATETIME_INPUT_FORMATS = ['%Y-%m-%d %H:%M:%S', '%Y-%m-%d %H:%M:%S.%f', '%Y-%m-%d %H:%M', '%Y-%m-%d', '%m/%d/%Y %H:%M:%S', '%m/%d/%Y %H:%M:%S.%f', '%m/%d/%Y %H:%M', '%m/%d/%Y', '%m/%d/%y %H:%M:%S', '%m/%d/%y %H:%M:%S.%f', '%m/%d/%y %H:%M', '%m/%d/%y']
DATE_FORMAT = 'N j, Y'
DATE_INPUT_FORMATS = ['%Y-%m-%d', '%m/%d/%Y', '%m/%d/%y', '%b %d %Y', '%b %d, %Y', '%d %b %Y', '%d %b, %Y', '%B %d %Y', '%B %d, %Y', '%d %B %Y', '%d %B, %Y']
DEBUG = True
DEBUG_PROPAGATE_EXCEPTIONS = False
DECIMAL_SEPARATOR = '.'
DEFAULT_CHARSET = 'utf-8'
DEFAULT_CONTENT_TYPE = 'text/html'
DEFAULT_EXCEPTION_REPORTER_FILTER = 'django.views.debug.SafeExceptionReporterFilter'
DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
DEFAULT_FROM_EMAIL = 'webmaster@localhost'
DEFAULT_INDEX_TABLESPACE = ''
DEFAULT_RENDERER_CLASSES = "('rest_framework.renderers.JSONRenderer', 'rest_framework.renderers.BrowsableAPIRenderer')"
DEFAULT_TABLESPACE = ''
DISALLOWED_USER_AGENTS = []
DRF_FIREBASE_AUTH = {'FIREBASE_SERVICE_ACCOUNT_KEY': '********************', 'ALLOW_ANONYMOUS_REQUESTS': True}
EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
EMAIL_HOST = 'localhost'
EMAIL_HOST_PASSWORD = '********************'
EMAIL_HOST_USER = ''
EMAIL_PORT = 25
EMAIL_SSL_CERTFILE = None
EMAIL_SSL_KEYFILE = '********************'
EMAIL_SUBJECT_PREFIX = '[Django] '
EMAIL_TIMEOUT = None
EMAIL_USE_LOCALTIME = False
EMAIL_USE_SSL = False
EMAIL_USE_TLS = False
FILE_CHARSET = 'utf-8'
FILE_UPLOAD_DIRECTORY_PERMISSIONS = None
FILE_UPLOAD_HANDLERS = ['django.core.files.uploadhandler.MemoryFileUploadHandler', 'django.core.files.uploadhandler.TemporaryFileUploadHandler']
FILE_UPLOAD_MAX_MEMORY_SIZE = 10485760
FILE_UPLOAD_PERMISSIONS = None
FILE_UPLOAD_TEMP_DIR = None
FIRST_DAY_OF_WEEK = 0
FIXTURE_DIRS = []
FORCE_SCRIPT_NAME = None
FORMAT_MODULE_PATH = None
FORM_RENDERER = 'django.forms.renderers.DjangoTemplates'
IGNORABLE_404_URLS = []
INSTALLED_APPS = ['django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.messages', 'django.contrib.staticfiles', 'django.contrib.gis', 'django.contrib.postgres', 'django_extensions', 'api.apps.ApiConfig', 'colorfield', 'rest_framework', 'rest_framework_gis', 'drf_firebase_auth', 'ordered_model', 'silk', 'debug_toolbar', 'django.contrib.admin', 'django.contrib.admindocs']
INTERNAL_IPS = ['172.20.0.1', '127.0.0.1']
LANGUAGES = [('af', 'Afrikaans'), ('ar', 'Arabic'), ('ast', 'Asturian'), ('az', 'Azerbaijani'), ('bg', 'Bulgarian'), ('be', 'Belarusian'), ('bn', 'Bengali'), ('br', 'Breton'), ('bs', 'Bosnian'), ('ca', 'Catalan'), ('cs', 'Czech'), ('cy', 'Welsh'), ('da', 'Danish'), ('de', 'German'), ('dsb', 'Lower Sorbian'), ('el', 'Greek'), ('en', 'English'), ('en-au', 'Australian English'), ('en-gb', 'British English'), ('eo', 'Esperanto'), ('es', 'Spanish'), ('es-ar', 'Argentinian Spanish'), ('es-co', 'Colombian Spanish'), ('es-mx', 'Mexican Spanish'), ('es-ni', 'Nicaraguan Spanish'), ('es-ve', 'Venezuelan Spanish'), ('et', 'Estonian'), ('eu', 'Basque'), ('fa', 'Persian'), ('fi', 'Finnish'), ('fr', 'French'), ('fy', 'Frisian'), ('ga', 'Irish'), ('gd', 'Scottish Gaelic'), ('gl', 'Galician'), ('he', 'Hebrew'), ('hi', 'Hindi'), ('hr', 'Croatian'), ('hsb', 'Upper Sorbian'), ('hu', 'Hungarian'), ('hy', 'Armenian'), ('ia', 'Interlingua'), ('id', 'Indonesian'), ('io', 'Ido'), ('is', 'Icelandic'), ('it', 'Italian'), ('ja', 'Japanese'), ('ka', 'Georgian'), ('kab', 'Kabyle'), ('kk', 'Kazakh'), ('km', 'Khmer'), ('kn', 'Kannada'), ('ko', 'Korean'), ('lb', 'Luxembourgish'), ('lt', 'Lithuanian'), ('lv', 'Latvian'), ('mk', 'Macedonian'), ('ml', 'Malayalam'), ('mn', 'Mongolian'), ('mr', 'Marathi'), ('my', 'Burmese'), ('nb', 'Norwegian Bokmål'), ('ne', 'Nepali'), ('nl', 'Dutch'), ('nn', 'Norwegian Nynorsk'), ('os', 'Ossetic'), ('pa', 'Punjabi'), ('pl', 'Polish'), ('pt', 'Portuguese'), ('pt-br', 'Brazilian Portuguese'), ('ro', 'Romanian'), ('ru', 'Russian'), ('sk', 'Slovak'), ('sl', 'Slovenian'), ('sq', 'Albanian'), ('sr', 'Serbian'), ('sr-latn', 'Serbian Latin'), ('sv', 'Swedish'), ('sw', 'Swahili'), ('ta', 'Tamil'), ('te', 'Telugu'), ('th', 'Thai'), ('tr', 'Turkish'), ('tt', 'Tatar'), ('udm', 'Udmurt'), ('uk', 'Ukrainian'), ('ur', 'Urdu'), ('vi', 'Vietnamese'), ('zh-hans', 'Simplified Chinese'), ('zh-hant', 'Traditional Chinese')]
LANGUAGES_BIDI = ['he', 'ar', 'fa', 'ur']
LANGUAGE_CODE = 'en-us'
LANGUAGE_COOKIE_AGE = None
LANGUAGE_COOKIE_DOMAIN = None
LANGUAGE_COOKIE_NAME = 'django_language'
LANGUAGE_COOKIE_PATH = '/'
LOCALE_PATHS = []
LOGGING = {}
LOGGING_CONFIG = 'logging.config.dictConfig'
LOGIN_REDIRECT_URL = '/accounts/profile/'
LOGIN_URL = '/accounts/login/'
LOGOUT_REDIRECT_URL = None
MANAGERS = []
MEDIA_ROOT = ''
MEDIA_URL = ''
MESSAGE_STORAGE = 'django.contrib.messages.storage.fallback.FallbackStorage'
MIDDLEWARE = ['django.middleware.common.CommonMiddleware', 'django.contrib.sessions.middleware.SessionMiddleware', 'django.contrib.auth.middleware.AuthenticationMiddleware', 'django.contrib.messages.middleware.MessageMiddleware', 'debug_toolbar.middleware.DebugToolbarMiddleware', 'silk.middleware.SilkyMiddleware']
MIGRATION_MODULES = {}
MONTH_DAY_FORMAT = 'F j'
NUMBER_GROUPING = 0
PASSWORD_HASHERS = '********************'
PASSWORD_RESET_TIMEOUT_DAYS = '********************'
PREPEND_WWW = False
REST_FRAMEWORK = {'DEFAULT_PERMISSION_CLASSES': ['rest_framework.permissions.IsAuthenticatedOrReadOnly'], 'DEFAULT_RENDERER_CLASSES': ('rest_framework.renderers.JSONRenderer', 'rest_framework.renderers.BrowsableAPIRenderer'), 'DEFAULT_AUTHENTICATION_CLASSES': ['drf_firebase_auth.authentication.FirebaseAuthentication']}
ROOT_URLCONF = 'chron.urls'
SECRET_KEY = '********************'
SECURE_BROWSER_XSS_FILTER = False
SECURE_CONTENT_TYPE_NOSNIFF = False
SECURE_HSTS_INCLUDE_SUBDOMAINS = False
SECURE_HSTS_PRELOAD = False
SECURE_HSTS_SECONDS = 0
SECURE_PROXY_SSL_HEADER = None
SECURE_REDIRECT_EXEMPT = []
SECURE_SSL_HOST = None
SECURE_SSL_REDIRECT = False
SERVER_EMAIL = 'root@localhost'
SESSION_CACHE_ALIAS = 'default'
SESSION_COOKIE_AGE = 1209600
SESSION_COOKIE_DOMAIN = None
SESSION_COOKIE_HTTPONLY = True
SESSION_COOKIE_NAME = 'sessionid'
SESSION_COOKIE_PATH = '/'
SESSION_COOKIE_SAMESITE = 'Lax'
SESSION_COOKIE_SECURE = False
SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
SESSION_EXPIRE_AT_BROWSER_CLOSE = False
SESSION_FILE_PATH = None
SESSION_SAVE_EVERY_REQUEST = False
SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
SETTINGS_MODULE = 'chron.settings'
SHORT_DATETIME_FORMAT = 'm/d/Y P'
SHORT_DATE_FORMAT = 'm/d/Y'
SIGNING_BACKEND = 'django.core.signing.TimestampSigner'
SILENCED_SYSTEM_CHECKS = []
SILKY_PYTHON_PROFILER = True
SILKY_PYTHON_PROFILER_BINARY = True
SILKY_PYTHON_PROFILER_RESULT_PATH = 'profiles/'
STATICFILES_DIRS = []
STATICFILES_FINDERS = ['django.contrib.staticfiles.finders.FileSystemFinder', 'django.contrib.staticfiles.finders.AppDirectoriesFinder']
STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
STATIC_ROOT = '/src/static'
STATIC_URL = '/static/'
TEMPLATES = [{'BACKEND': 'django.template.backends.django.DjangoTemplates', 'DIRS': [], 'APP_DIRS': True, 'OPTIONS': {'context_processors': ['django.template.context_processors.debug', 'django.template.context_processors.request', 'django.contrib.auth.context_processors.auth', 'django.contrib.messages.context_processors.messages']}}]
TEST_NON_SERIALIZED_APPS = []
TEST_RUNNER = 'django.test.runner.DiscoverRunner'
THOUSAND_SEPARATOR = ','
TIME_FORMAT = 'P'
TIME_INPUT_FORMATS = ['%H:%M:%S', '%H:%M:%S.%f', '%H:%M']
TIME_ZONE = 'UTC'
USE_I18N = True
USE_L10N = True
USE_THOUSAND_SEPARATOR = False
USE_TZ = True
USE_X_FORWARDED_HOST = False
USE_X_FORWARDED_PORT = False
WSGI_APPLICATION = 'chron.wsgi.application'
X_FRAME_OPTIONS = 'SAMEORIGIN'
YEAR_MONTH_FORMAT = 'F Y'


You're seeing this error because you have DEBUG = True in your
Django settings file. Change that to False, and Django will
display a standard page generated by the handler for this status code.

```

Reply:
That is the object of the response, there is no way I could access the encoded value. Or I missed it somehow
![image](https://user-images.githubusercontent.com/61106/73246198-4bcc0c00-41ae-11ea-92e0-381cc7396d98.png)

Reply:
Did you try running a production build?

Reply:
Actually, that just gave me an error 500. I'll look into this.